go.mod

```diff
@@ -40,9 +40,7 @@ require (
 	cloud.google.com/go/kms v1.1.0
 	cloud.google.com/go/pubsub v1.16.0
 	cloud.google.com/go/storage v1.21.0
-	github.com/Azure/azure-pipeline-go v0.2.3 // indirect
```

Comment:
Can this be removed entirely from our dependencies?

Reply:
Yup. All the changes to go.mod, go.sum, and distdir_files.bzl trickle out of following the [directions ](https://github.com/cockroachdb/cockroach/tree/master/build) of updating dependencies. The only actual go code I modified which caused all of this is modifying the imports in `azure_storage.go`.